source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js

```diff
@@ -126,7 +126,7 @@
         var self = this;
         this.userView = data.UserView;
         this.userLogged = data.UserLogged;
-        this.hasAverageColumn = !data.Evaluation.Finished && (this.userView == 1 || this.userView == 3);
```

Comment:
1 and 3 are Responsible and Company. The average just was shown for responsible and company, but now this column is shown for evaluators too, as is mentioned in the document